extensions/wikia/ArticleVideo/fonts/ooyala-slick-type.svg

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg xmlns="http://www.w3.org/2000/svg">
```

Comment:
Let's check if we can use only the SVG and not include the fonts. Then we'll be able to use DS icons easily.